### DISCUSSION TITLE
**Issue Request for Support with Azuracast Radio Software** Crossfade?

### DISCUSSION THREAD

**Discussion by WishBoneBoy007** [AzuraCast/AzuraCast#6659] [Q&A]
*Issue Description:**

1. **Sample 1:** http://sndup.net/z4pq 
   - Problem: The final seconds of a song repeat itself and crossfade into the next song with a crossfade duration of 0.1. When set to a value higher than this, there are significant gaps of silence.
   
2. **Sample 2:** http://sndup.net/sm32
   - Problem: Transition from a song to an ID is skipping part of the ID content. It should say 'Beats, By Bike It Now,' but it skips part of this ID before transitioning into the next song. This issue seems to occur with the majority, if not all, IDs.

3. **Sample 3:** http://sndup.net/cwr3
   - Problem: Similar to Sample 2, transitioning from the ending of a song into an ID results in skipping part of the ID content before moving to the next song.

These issues are causing significant disruptions for my stations and are negatively affecting the user experience.

**Environment:**

- Hosting: Linode
- Azuracast Version: Latest stable version
- Server Resources: Adequate CPU and RAM (60% CPU usage, not reaching any near use of 8GB of RAM)
- Configuration: Using Master_Me Music General setting
- Additional Info: This issue persists even when using a licensed Stereo Tool plugin.

**Attempts to Resolve:**

I have made several attempts to resolve this issue, including adjusting crossfade settings and bit rates, but the problem still persists.

**Request for Help:**

I kindly request assistance from the Azuracast community to resolve these issues and restore the software's functionality to how it worked in previous versions when this problem did not occur.

Thank you in advance for your help and support.

**Cheers!**

**Comment by Sweetness2k** [comment]
We all have the same issues it's a LS issue. I just turn off my crossfading till they fix the issue but it's not a Azuracast issue.

**Comment by LordHelmchen666** [comment]
Do you have your files on the host machine or on a external mount? I had some problems with external mounted folder. 
Do you use Replaygain? Are all files tagged  with Replaygain values?
